local/docker-compose.yml

```diff
@@ -2,20 +2,15 @@ version: "2"
 
 services:
   conjur:
-    image: cyberark/conjur:0.1.0-stable
+    image: cyberark/conjur:0.3.0-stable
+    command: server -a default
     environment:
-      CONJUR_APPLIANCE_URL: http://localhost:3000
       DATABASE_URL: postgres://postgres@pg/postgres
-      CONJUR_ADMIN_PASSWORD: admin
-      PORT: 3000
-      RAILS_ENV:
-    ports: [ "3000:3000" ]
-    volumes: [ "./tmp:/app_tmp"]
-    entrypoint: |
-      bash -c "export CONJUR_DATA_KEY=\"`conjurctl data-key generate`\" && \
-               conjurctl db migrate && \
-               conjurctl account create default | tail -n1 | awk '{print $$NF}' > /app_tmp/api_key && \
-               conjurctl server"
+      CONJUR_DATA_KEY: 'WMfApcDBtocRWV+ZSUP3Tjr5XNU+Z2FdBb6BEezejIs='
```

Comment:
Does it not matter what this value gets set to?

Reply:
Not as far as I can tell ...